cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/DetailsInfo/index.js

```diff
@@ -64,6 +64,17 @@ export default class ProfileDetailViewDetailsInfo extends Component {
       );
     }
 
+    const propertyToLabelMap = {};
```

Comment:
Shouldn't this have been declared with a `let` if you're modifying it?

Reply:
So `const` doesn't ensure immutability. 
```
const myObj = {};
myObj.properti1 = 'value1';
```
is ok but,
```
const myObj = {};
myObj = {
  property1: 'value1'
}
```
will cause issue.